admin/app/com/lucidchart/piezo/admin/controllers/TriggerFormHelper.scala

```diff
@@ -127,20 +137,62 @@ class TriggerFormHelper(scheduler: Scheduler) extends JobDataHelper {
       "jobGroup" -> nonEmptyText(),
       "jobName" -> nonEmptyText(),
       "description" -> text(),
-      "simple" -> optional(mapping(
-        "repeatCount" -> number(),
-        "repeatInterval" -> number()
-      )(simpleScheduleFormApply)(simpleScheduleFormUnapply)),
-      "cron" -> optional(mapping(
-        "cronExpression" -> nonEmptyText().verifying(validCronExpression)
-      )(cronScheduleFormApply)(cronScheduleFormUnapply)),
+      "simple" -> optional(
+        mapping(
+          "repeatCount" -> number(),
+          "repeatInterval" -> number(),
+        )(simpleScheduleFormApply)(simpleScheduleFormUnapply),
+      ),
+      "cron" -> optional(
+        mapping(
+          "cronExpression" -> nonEmptyText().verifying(validCronExpression),
+        )(cronScheduleFormApply)(cronScheduleFormUnapply),
+      ),
       "job-data-map" -> jobDataMap,
       "triggerMonitoringPriority" -> nonEmptyText(),
-      "triggerMaxErrorTime" -> number()
-    )(triggerFormApply)(triggerFormUnapply).verifying("Job does not exist", fields => {
-      scheduler.checkExists(fields._1.getJobKey)
-    }).verifying("Max time between successes must be greater than 0", fields => {
-      fields._3 > 0
-    })
+      "triggerMaxErrorTime" -> of(MaxSecondsBetweenSuccessesFormatter).verifying(greaterThan(0)),
+    )(triggerFormApply)(triggerFormUnapply)
+      .verifying(
+        "Job does not exist",
+        fields => {
+          scheduler.checkExists(fields._1.getJobKey)
+        },
+      )
+      .verifying(
+        "Max time between successes must be greater than 0",
+        fields => {
+          fields._3 > 0
+        },
+      ),
```

Comment:
I had it written this way originally. The only downside is that the error message is at the top of the form instead of right beneath the field that actually needs to be changed. I feel like it's easier to read if the error message is close to what needs to be changed.

Reply:
hmm, that's unfortunate. I'm not sure.

Reply:
I wonder if you specify `"triggerMaxErrorTime"` as the name of the constraint, if it will apply to that field?

Reply:
This didn't end up working. Would you be ok if I left it the way I have it now?